src/MongoDB.Driver.Core/Core/Operations/AggregateOperation.cs

```diff
@@ -343,16 +344,13 @@ internal BsonDocument CreateCommand(ConnectionDescription connectionDescription,
                 { "readConcern", readConcern, readConcern != null }
             };
 
-            if (Feature.AggregateCursorResult.IsSupported(connectionDescription.ServerVersion))
+            var useCursor = _useCursor.GetValueOrDefault(true) || connectionDescription.ServerVersion >= new SemanticVersion(3, 5, 0);
```

Comment:
Use released version, e.g. `3, 6, 0`

Reply:
I will update this to use 3.6.0.

Reply:
Done

Reply:
Done in scope of https://github.com/rstam/mongo-csharp-driver/pull/142